iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/RemoteReactivePowerAdderImpl.java

```diff
@@ -34,18 +38,22 @@ protected RemoteReactivePowerControl createExtension(final Generator extendable)
 
     @Override
     public RemoteReactivePowerControlAdder withTargetQ(double targetQ) {
+        ValidationUtil.checkReactivePowerSetpoint((Validable) super.extendable, VALIDABLE_TYPE_DESCRIPTION, targetQ, enabled);
         this.targetQ = targetQ;
         return this;
     }
 
     @Override
     public RemoteReactivePowerControlAdder withRegulatingTerminal(Terminal regulatingTerminal) {
+        ValidationUtil.checkRegulatingTerminal((Validable) super.extendable, VALIDABLE_TYPE_DESCRIPTION, regulatingTerminal, enabled, super.extendable.getNetwork());
         this.regulatingTerminal = regulatingTerminal;
         return this;
     }
 
     @Override
     public RemoteReactivePowerControlAdder withEnabled(boolean enabled) {
+        ValidationUtil.checkReactivePowerControl((Validable) super.extendable, regulatingTerminal,
+            targetQ, enabled, super.extendable.getNetwork());
```

Comment:
Said it above but checks must be in `createExtension`, not in intermediary methods for adders

Reply:
Done